lib/widget/image.dart

```diff
@@ -85,19 +86,14 @@ class ImageState extends WidgetState<EnsembleImage> {
         // image binding is tricky. When the URL has not been resolved
         // the image will throw exception. We have to use a permanent placeholder
         // until the binding engages
-        return Image.network(
-            widget._controller.source,
-            width: widget._controller.width?.toDouble(),
+        return 
```

Comment:
@Moeez1999 we need to use CachedNetworkImage if the cache property is set to true. Default is true. Please read the instructions. You are using the CachedImage always without checking the property

Reply:
also what about svgImage?

Reply:
I have done the cache one for network images and for the SVG i am working on it when it will be completed i will push that